frontend/src/views/WheresMyGene/components/Main/index.tsx

```diff
@@ -271,27 +295,34 @@ export default function WheresMyGene(): JSX.Element {
           setIsScaled={setIsScaled}
         />
       </SideBar>
-      {cellInfoCellType && tissuesByID && (
-        <SideBar
-          label={`${cellInfoCellType.cellType.name}`}
-          SideBarWrapperComponent={SideBarWrapper}
-          SideBarPositionerComponent={SideBarPositioner}
-          SideBarOpenButtonWrapperComponent={GeneSideBarOpenButtonWrapper}
-          position={Position.RIGHT}
-          testId="cell-type-details-panel"
-          disabled={false}
-          forceOpen={forceOpen}
-          wmgSideBar
-          width={CELL_INFO_SIDEBAR_WIDTH_PX}
-          truncatedLabel={`${tissuesByID[cellInfoCellType.tissueID].name} - ${
-            cellInfoCellType.cellType.name
-          }`}
-        >
+      {cellInfoCellType && tissuesByID ? (
+        <RightSideBar>
           <CellInfoBar
+            generateGeneInfo={generateGeneInfo}
             cellInfoCellType={cellInfoCellType}
             tissueInfo={tissuesByID[cellInfoCellType.tissueID]}
+            handleClose={handleCloseRightSideBar}
+            title={`${cellInfoCellType.cellType.name}`}
           />
-        </SideBar>
+
+          {geneInfoGene && (
+            <GeneInfoBar
+              geneInfoGene={geneInfoGene}
+              handleClose={handleCloseGeneInfoSideBar}
+              title={`${geneInfoGene}`}
+            />
+          )}
+        </RightSideBar>
+      ) : (
+        geneInfoGene && (
```

Comment:
if only gene info is populated then render the gene info panel at full length (not split)